spark/wordcount/pom.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>cloud-bigtable-dataproc-spark-wordcount</name>
+    <groupId>com.example.bigtable.spark.wordcount</groupId>
+    <artifactId>cloud-bigtable-dataproc-spark-wordcount</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1</version>
+
+    <properties>
+        <bigtable.version>0.9.7.1</bigtable.version>
+        <hbase.version>1.3.1</hbase.version>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <spark.version>2.1.1</spark.version>
+        <scala.version>2.11.0</scala.version>
+        <scalatest.version>3.0.3</scalatest.version>
+        <skipTests>true</skipTests>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-core_2.11</artifactId>
+            <version>${spark.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-sql_2.11</artifactId>
+            <version>${spark.version}</version>
+            <!---<scope>provided</scope>-->
+        </dependency>
+        <dependency>
+            <groupId>com.google.cloud.bigtable</groupId>
+            <artifactId>bigtable-hbase-1.1</artifactId>
```

Comment:
The bigtable-hbase-1.x-hadoop artifact should be used here, if you upgrade to the 1.0.0-pre2 version